bitcoin/src/blockdata/witness.rs

```diff
@@ -194,25 +233,29 @@ impl Witness {
     pub fn clear(&mut self) {
         self.content.clear();
         self.witness_elements = 0;
-        self.last = 0;
-        self.second_to_last = 0;
+        self.indices_start = 0;
     }
 
     /// Push a new element on the witness, requires an allocation
     pub fn push<T: AsRef<[u8]>>(&mut self, new_element: T) {
         let new_element = new_element.as_ref();
```

Comment:
Unrelated: we could de-monomorphise this.

Reply:
I wonder if clippy could lint about this?

Reply:
https://github.com/rust-lang/rust-clippy/issues/1375